MainModule/Server/Dependencies/Assets/BotBrain/init.server.lua

```diff
@@ -153,11 +153,12 @@ getPath = function()
 		if target then
 			targetPos = target.CFrame
 		else
-			targetPos = CFrame.new(props.PatrolZone+Vector3.new(math.random(-props.PatrolDist,props.PatrolDist),0,math.random(-props.PatrolDist,props.PatrolDist)))
-			task.wait(0.1)
+			targetPos = CFrame.new(props.PatrolZone + Vector3.new(math.random(-props.PatrolDist, props.PatrolDist),0,math.random(-props.PatrolDist, props.PatrolDist)))
+			task.wait(math.random())
```

Comment:
Why was this changed to math.random from 0.1

Reply:
So it waits a random delay between 0 and 1. Adds more randomness and also makes the bots more more realistically because 0.1 is shorter compared to math.random() on average which causes the bots to jolt back and fort.

Reply:
That is on average, though – if the problem is jolting why not set a minimum of 0.1 and a maximum of 1 or 0.5